ports/libfaketime/CMakeLists.txt

```
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(faketime VERSION 0.9.10)
```

```suggestion
project(faketime)
```

The version is not needed here. You don't have to change this now (to avoid restarting CI), just keep it in mind when upstreaming the port.